tools/gfx/d3d12/render-d3d12.cpp

```diff
@@ -3792,6 +3829,9 @@ class D3D12Device : public RendererBase
     bool m_nvapi = false;
 };
 
+uint32_t D3D12Device::BufferResourceImpl::id = 0;
```

Comment:
As mentioned, you can delete this and declare it as a local static variable in `getSharedHandle`.